crypto/bio/bss_dgram.c

```diff
@@ -636,6 +622,7 @@ static long dgram_ctrl(BIO *b, int cmd, long num, void *ptr)
         {
             struct timeval *tv = (struct timeval *)ptr;
             int timeout = tv->tv_sec * 1000 + tv->tv_usec / 1000;
```

Comment:
```
int timeout = (int)ossl_time2ms(ossl_time_from_timeval((struct timeval *)ptr));

Reply:
This comment seems to be outstanding

Reply:
We're converting from a timeval to milliseconds, both are known and the conversion is easy. I simply don't see any need to artificially go via an OSSL_TIME for this.